CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
+   * Kubernetes events for monitoring [node auto-repair](https://learn.microsoft.com/en-us/azure/aks/node-auto-repair) actions are now available for your AKS cluster. You can ingest these events and create alerts following the same [process as other Kubernetes events](https://learn.microsoft.com/azure/aks/events?tabs=azure-cli).
+   * AKS [Kubernetes patch versions](https://kubernetes.io/releases/patch-releases/) 1.29.12, 1.29.13, 1.30.8, 1.30.9, 1.31.4, and 1.31.5 are now available.
+
+* Preview Features:
+   * You can use the `EnableCiliumNodeSubnet' feature in preview to [create Cilium node subnet clusters](https://learn.microsoft.com/azure/aks/azure-cni-powered-by-cilium#option-3-assign-ip-addresses-from-the-node-subnet-preview) using Azure CNI Powered by Cilium.
+   * [Free control plane metrics](https://learn.microsoft.com/azure/aks/control-plane-metrics-monitor) are now available in preview to monitor the API server and ETCD components, which will be automatically collected by default on your cluster.
+
+* Bug Fixes:
+   * Resolved an [issue](https://github.com/Azure/AKS/issues/4662) with Istio service mesh add-on where having multiple operations with the Lua EnvoyFilter (e.g. adding the Lua filter to call an external service and specifying the cluster referenced by Lua code) was not allowed.
+   * Fixed a bug in Azure CNI Pod Subnet Static Block Allocation mode with Cilium which caused incorrect iptables rules, leading to pod connectivity failures to DNS and IMDS.
+   * Resolved an [issue](https://github.com/Azure/azure-sdk-for-go/issues/24109) where the updated Azure Table client mishandled untyped numbers, causing static block agent pools to be misidentified as dynamic and leading to operation failures.
+   * Fixed a bug in Azure Kubernetes Fleet Manager hubful resource groups by truncating the name to avoid issues with long generated managed resource group names breaking the maximum length of resource groups.
+
+* Behavior Changes:
+   * [Horizontal Pod Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) introduced for `ama-metrics replicaset pod` in the [Azure Monitor managed service for Prometheus](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/prometheus-metrics-overview) add-on. More details about the configuration of the Horizontal Pod Autoscaler can be found [here](https://github.com/Azure/prometheus-collector/blob/main/internal/docs/HPA.md).
+
+* Component Updates:
+   * Starting with Kubernetes v1.32, node subnet mode will be installed via a DaemonSet (CNS) instead of AgentBaker. Customers will now see this DaemonSet running in their clusters, aligning node subnet mode with PodSubnet and Overlay CNI for a more uniform architecture and faster security updates.
```

Comment:
need to remove 'AgentBaker'

Reply:
can we mention the exact name of the daemonset? iirc it should be azure-cns. if yes, then it should be "Starting with Kubernetes v1.32, node subnet mode will be installed via `azure-cns` DaemonSet, allowing for faster security updates"

Reply:
also this may need to be under behavior change section. need to see the PR, will sync offline